qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/interactor/PlacementInteractorImpl.kt

```diff
@@ -97,15 +97,17 @@ internal class PlacementInteractorImpl(
 
   private fun buildPlacement(placementContent: PlacementContentModel?): PlacementImpl? {
     return if (placementContent != null) {
+      val impressionUrl = placementContent.callbacks.impression ?: ""
```

Comment:
Why empty string is better sign of lack of URL, than null?

Null seems to be more natural.

Reply:
I wasn't sure how changing these properties to nullables would work with ReactNative bridge. As I'm not very familiar with it and we're rolling out this change as a hotfix to customers who already received placement release I wanted to do least risky change here.